server/src/main/resources/transport/upper_bounds/9.2.csv

```diff
@@ -1 +1 @@
-security_stats_endpoint,9168000
+inference_api_openai_embeddings_headers,9169000
```

Comment:
This seems to have snuck in somehow

Reply:
And the [test failure](https://gradle-enterprise.elastic.co/s/ht3z6rp6kbjga/failures#1) is about whatever this thing is

Reply:
@schase-es there was a discussion on [slack](https://elastic.slack.com/archives/C09GG7A3CQ7/p1758730220363479) if you're curious.